src/Draco.Compiler/Api/CodeCompletion/CompletionContext.cs

```diff
@@ -0,0 +1,32 @@
+namespace Draco.Compiler.Api.CodeCompletion;
+
+/// <summary>
+/// Represents a code context of a completion (For example if the completion is part of expression/ModuleImport/...).
+/// </summary>
+public enum CompletionContext
+{
+    /// <summary>
+    /// Part of an expression (can be also keyword if its usable in expressions, for example 'or').
+    /// </summary>
+    ExpressionContent,
+
+    /// <summary>
+    /// Expression where only valid suggestions are types.
+    /// </summary>
+    Type,
+
+    /// <summary>
+    /// Member access expression.
```

Comment:
```suggestion
    /// Inside some kind of member access syntax.
```

Reply:
Does this mean that I'll also receive `Expression/Type/Import` along with this context so I know what to suggest exactly?